Erik/LayoutEngine.swift

```diff
@@ -217,8 +217,8 @@ extension WebKitLayoutEngine {
                         completionHandler?(object, e) // must not be called
                         return
                     }
-                    
-                    let result = self.wait(key, timeout: DispatchTimeInterval.seconds(self.javaScriptWaitTime).fromNow)
+                  
+                    let result = self.wait(key, timeout: DispatchTime.now() + DispatchTimeInterval.seconds(self.javaScriptWaitTime))
                     
```

Comment:
previous code seems to work also on OSX for me but maybe because I use an other repository for BrightFuture (pod 'BrightFutures', :git => 'http://github.com/oysta/BrightFutures.git', :branch => 'xcode8b6')
In fact it's a BrightFuture extension, so I keep your code (less dependency on other project)
```swift
public extension DispatchTimeInterval {
    public var fromNow: DispatchTime {
        return DispatchTime.now() + self
    }
}
```